frontend/src/components/Receipt/LineItemsTableMobile.tsx

```
@@ -51,93 +54,118 @@ export default function ReceiptLineItemsTableMobile({
-          initial={{ opacity: 0 }}
-          animate={{ opacity: 1 }}
-          transition={{ delay: 0.05 }}
-        >
+      {line_items.map((item) => {
+        const showReducedDetails = assignmentItemId === item.id;
+        const showReducedAssignments = editItemId === item.id;
+
+        return (
           <LineItemCard
             selected={editItemId === item.id || assignmentItemId === item.id}
           >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)